crates/cli-tools/src/changelog.rs

```diff
@@ -16,13 +16,14 @@ use core::str;
 use std::collections::BTreeMap;
 use std::fmt::{Display, Write};
 use std::io::BufRead;
+use std::str::FromStr;
 
 use anyhow::{anyhow, bail, ensure, Context, Result};
 use semver::Version;
 use tokio::process::Command;
-use wasefire_cli_tools::{cmd, fs};
+use crate::{cmd, fs};
 
-#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, clap::ValueEnum)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
```

Comment:
Why not use `ValueEnum` anymore?

Reply:
I was thinking so we didn't need to add clap as a dependency (FromString logic simple enough).

However, since we are hiding changelog module behind a feature anways, might as well include clap. I've reverted back to ValueEnum, no FromStr impl.

Reply:
I see. But we already have it as an optional dependency for the `action` feature. So we can also enable it with `changelog`.

Reply:
Yep, agreed. That's what we will do now.

(Before changelog was behind a feature, I was concerned it wasn't worth always adding this dependency.)